app/queries/hackney_api.rb

```diff
@@ -13,11 +13,11 @@ def get_property(property_reference:)
   end
 
   def create_repair(repair_params)
-    @json_api.post('repairs', repair_params)
+    @json_api.post('v1/repairs', repair_params)
```

Comment:
We can't do it now, but do you think we should ultimately make the version part of the API root?

Reply:
I completely agree :+1: 

Thinking about this, I was imagining scenarios where you'd want to use `v2` of a particular endpoint to get additional data back, but `v1` of everything else - but in reality, new versions  should maintain compatibility as much as possible.